espaloma/graphs/graph.py

```diff
@@ -44,7 +44,8 @@ class Graph(BaseGraph):
 
     """
 
-    def __init__(self, mol=None, homograph=None, heterograph=None):
+    def __init__(self, mol=None, homograph=None, heterograph=None,
```

Comment:
this modification is a bit too aggressive as we need to supply another arguments at the very beginning stage of graph definition. can we not touch the `esp.graph` part and move this to another utility function where we take a `esp.Graph` and modify in-place?